nativeApp/src/lib/ui/forms/Dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+import styled from 'styled-components/native'
+import { Item as I, Picker, Label as L } from 'native-base';
+import { TextInputProps } from 'react-native';
+import { Forms } from './enums';
+import { MaterialIcons } from '@expo/vector-icons';
+import { ColoursEnum } from '../colours';
+
+
+/*
+ * Types
+ */
+type Props = {
+  label: string;
+  options: {id: number; name: string}[];
+  selectedValue?: string;
+  onValueChange?: () => void;
+} & TextInputProps
```

Comment:
I don't think we should use this form to extend existing types, because it can appear fine on definition, but cause problems on use (in the case where we're overwriting props that already exist)

Example:
```ts
interface One {
  prop: string;
}

type Two = { // this definition looks OK
  prop: number;
} & One;

type Three = One & { // this definition looks OK
  prop: boolean;
}

const x: Two = {
  prop: 1, // Type 'number' is not assignable to type 'never'
};

const y: Three = {
  prop: true, // Type 'number' is not assignable to type 'never'
}
```

Instead if we use `interface` we can see the issue immediately:
```ts
interface Four extends One { // Interface 'Four' incorrectly extends interface 'One'.
  prop: boolean;
}
```